app/Exceptions/GistNotFoundException.php

```diff
@@ -2,7 +2,9 @@
 
 namespace App\Exceptions;
 
-class GistNotFoundException extends \Exception
+use Throwable;
+
+class GistNotFoundException extends Throwable
```

Comment:
Hm, this is probably from the other PR you extended, but I'm inclined to keep an exception an exception. Would you mind reverting this?